moveit_planners/ompl/ompl_interface/src/ompl_interface.cpp

```diff
@@ -232,27 +224,11 @@ void ompl_interface::OMPLInterface::loadPlannerConfigurations()
     pconfig[default_pc.name] = default_pc;
 
     // get parameters specific to each planner type
-    XmlRpc::XmlRpcValue config_names;
-    if (nh_.getParam(group_name + "/planner_configs", config_names))
+    std::vector<std::string> config_names;
+    if (node_->get_parameter(group_name + ".planner_configs", config_names))
```

Comment:
Are parameters separated by "/" or "." in ros2? This PR seems to use both

Reply:
It's ".". As pointed out in the description this PR only contains straight migration steps, I'm providing a thorough parameter refactoring in a follow-up PR. I can change these for cleanliness, though.